frontend/src/components/user-profile/user-profile.tsx

```diff
@@ -172,6 +204,23 @@ const UserPage: React.FC = () => {
           </span>
         )}
       </div>
+      {isOwnPage && <div className={styles.favoriteEpisodes}>
+        <h2 className={styles.podcastsByUserTitle}>Favorite episodes:</h2>
+        {favoriteEpisodes.length
```

Comment:
Maybe if will be better if put this to `hasFavouriteEpisodes`?

Reply:
agreed